src/contract.rs

```diff
@@ -254,6 +279,19 @@ fn query_token_info<S: ReadonlyStorage>(storage: &S) -> QueryResult {
     })
 }
 
+fn query_token_config<S: ReadonlyStorage>(storage: &S) -> QueryResult {
+    let config = ReadonlyConfig::from_storage(storage);
+    let constants = config.constants()?;
+
+    to_binary(&QueryAnswer::TokenConfig {
+        public_total_supply: constants.total_supply_is_public,
+        deposit_enabled: constants.deposit_is_enabled,
+        redeem_enabled: constants.redeem_is_enabled,
+        mint_enabled: constants.mint_is_enabled,
+        burn_enabled: constants.burn_is_enabled,
+    })
+}
+
```

Comment:
Should we add this to the specification, or should we just think of it as a reference-impl specific extension? I'm leaning towards option B but I'm curious to hear your thoughts

Reply:
I would lean towards just reference implementation specific as well